framework/validators/InlineValidator.php

```diff
@@ -60,6 +60,18 @@ class InlineValidator extends Validator
     /**
      * @inheritdoc
      */
+    public function validateModel($model)
```

Comment:
Yes, I am not sure about this implementation and name of this method. Can be `validateCommon` or something like that. Want to hear your opinions and advices.

I separated it because it doesn't fit `validateAttributes` logic and flow. Can't think of better solution with not too much changes at the moment.